scipy/stats/stats.py

```diff
@@ -5303,13 +5335,15 @@ def ttest_1samp(a, popmean, axis=0, nan_policy='propagate'):
     Examples using axis and non-scalar dimension for population mean.
 
     >>> stats.ttest_1samp(rvs,[5.0,0.0])
-    (array([-0.68014479,  4.11038784]), array([  4.99613833e-01,   1.49986458e-04]))
+    (array([-0.68014479,  4.11038784]), array([  4.99613833e-01,
+    1.49986458e-04]))
```

Comment:
I guess a simple linebreak like this works, but it looks a bit messy in the later examples.  How about breaking it up, something like this:
```
>>> result = stats.ttest_1samp(rvs,[5.0,0.0])
>>> result.statistic
array([-0.68014479,  4.11038784])
>>> result.pvalue
array([  4.99613833e-01,   1.49986458e-04])
```